### PR TITLE
feat: remove posibility of editing e-mail

### LINKED ISSUE #11: Remove the possibility of editing the user and email
When a new user is registered, the form that appears to confirm the registration details should not allow the user to modify the fields loaded by default

### PR DESCRIPTION
**Pull request by lotharking**

### CODE REVIEW

File: docker-dev/docker-compose.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ services:
 
   keycloak:
     image: quay.io/keycloak/keycloak:23.0.5
-    command: start-dev --import-realm --http-relative-path=/auth
+    command: start-dev --import-realm --http-relative-path=/auth --features=preview
```
Comment (genaris) [inline docker-dev/docker-compose.yaml:72]: Why do we need to enable the preview features?

**Comment by genaris** [comment]
Don't forget to use semantic commit messages in PR title (e.g. feat: remove posibility of editing e-mail)

File: docker-dev/docker-compose.yaml (hunk 1 above)
Comment (lotharking) [reply docker-dev/docker-compose.yaml:72]: Yes, this feature allows you to control users because it is an additional feature.

Comment (genaris) [reply docker-dev/docker-compose.yaml:72]: Can you ellaborate more in regards to that? What do you mean by 'control users'?

Comment (lotharking) [reply docker-dev/docker-compose.yaml:72]: image

Comment (lotharking) [reply docker-dev/docker-compose.yaml:72]: This is the feature that allows the user to remove the ability to edit some specific data such as email in our case, without enabling it it is not possible to control it with the current version of keycloak

Comment (genaris) [reply docker-dev/docker-compose.yaml:72]: Fair enough!

**Review by genaris — APPROVED** [review]

**Pull request merged**